javascript/javascript3/week1/in_class_Exercises/dogfun.js

```diff
@@ -0,0 +1,90 @@
+// variables 
+const q1 = document.querySelector('#question1')
+const q2 = document.querySelector('#question2')
+const q3 = document.querySelector('#question3')
+const q4 = document.querySelector('#question4')
+
+let getRandomImage = () =>
+    fetch('https://dog.ceo/api/breeds/image/random')
+        .then((response) => response.json())
+
+// 1 get a random image
+function renderImage() {
+    getRandomImage().then((data) => {
+        const img = document.createElement('img')
+        img.src = data.message
+        q1.appendChild(img)
+    })
+}
+renderImage()
+
+
+
+//que2  get a new image after 2 ssec 
+function renderRandomImage() {
+    getRandomImage()
+        .then((data) => {
+            const img = q2.querySelector('img')
+            img.src = data.message
+        })
+}
+q2.appendChild(document.createElement('img'))
+setInterval(renderRandomImage, 4000);
+
+
+
+
+// que 3 Get the list of all breeds from https://dog.ceo/api/breeds/list/all
+const getdogBreed =
+    fetch('https://dog.ceo/api/breeds/list/all')
+        .then((response) => response.json())
+
+function renderDogBreads() {
```

Comment:
Also try to think about naming here. renderDogBreeds does more than just rendering